lagunita/lms/static/sass/_overrides.scss

```diff
@@ -1,487 +1,37 @@
-        }
-        a.help-modal-button {
-            &:link, &:visited {
-                font-family: $sans-serif;
-                font-weight: bold;
-                background-color: $dark-green;
-                color: $white;
-            }
-        }
-        a:hover {
-            background-color: $dark-green;
-            color: $white;
-        }
-    }
-    &:first-child {
-        margin-top: 0;
-    }
-}
-
-body.discussion {
-    #global-navigation {
-        .wrapper-header {
-            .nav-global {
-                display: none;
-            }
+        img {
+            height: 50px;
         }
     }
 }
 
-#seq_content {
-    font-size: 16px;
-    letter-spacing: normal;
-    color: $black;
-
-    .unit-title {
-        display: none;
-    }
-
-    .xmodule_HtmlModule {
-        h1 {
-            font-weight:700;
-            font-size: 2em;
-            text-transform: none;
-        }
-        h2 {
-            font-weight:700;
-            font-size: 1.5em;
-            text-transform: none;
-            line-height: 1.38em;
-            color: $black;
-            margin-top: 1.5em;
-            margin-bottom: .5em;
-        }
-        h3 {
-            font-size: 1.3em;
-            text-transform: none;
-            line-height: 1.38em;
-            margin-top: 1em;
-            margin-bottom: .3em;
-        }
-        h4 {
-            font-size: 1.1em;
-            text-transform: none;
-            line-height: 1.38em;
-            margin-top: 1em;
-            margin-bottom: .3em;
-        }
-        h5 {
-            font-size: 1em;
-            text-transform: none;
-            line-height: 1.38em;
-            margin-top: 1em;
-            margin-bottom: .3em;
-        }
-        h6 {
-            font-style: italic;
-            font-size: 1em;
-            text-transform: none;
-            line-height: 1.38em;
-            margin-top: 1em;
-            margin-bottom: .3em;
-        }
-
-        .blue {
-            color: #476bb2;
-        }
-
-        ul {
-            margin-left:25px;
-        }
-        ul>li {
-            list-style-type: disc;
-        }
-        ul ul>li {
-            list-style-type: circle;
-        }
-        ul ul ul>li {
-            list-style-type: square;
-        }
-        ul ul ul ul>li {
-            list-style-type: disc;
-        }
-        ul ul ul ul ul>li {
-            list-style-type: circle;
-        }
-        ul ul ul ul ul ul>li {
-            list-style-type: square;
-        }
-
-        ol {
-            margin-left:25px;
-        }
-        ol>li {
-            list-style-type: decimal;
-        }
-        ol ol>li {
-            list-style-type: lower-alpha;
-        }
-        ol ol ol>li {
-            list-style-type: lower-roman;
-        }
-        ol ol ol ol>li {
-            list-style-type: decimal;
-        }
-        ol ol ol ol ol>li {
-            list-style-type: lower-alpha;
-        }
-        ol ol ol ol ol ol>li {
```

Comment:
Isn't this the custom styling we recently merged to override defaults in platform?

Reply:
yep, so this needs to come back

Reply:
and in fact the stuff around it is Grace's stuff, so that needs to come back too

Reply:
back, but in separate file